forms-flow-bpm/src/main/java/org/camunda/bpm/extension/commons/connector/HTTPServiceInvoker.java

```diff
@@ -68,8 +67,8 @@ private String getServiceId(String url) {
 			return BPM_ACCESS_HANDLER;
 		} else if (isUrlValid(url, fetchUrlFromProperty(ANALYSIS_URL))) {
 			return TEXT_ANALYZER_ACCESS_HANDLER;
-		} else if (enableCustomSubmission) {
-			return CUSTOM_SUBMISSION_ACCESS_HANDLER;
+		} else if (enableCustomSubmission && StringUtils.contains(url, "/submission")) {
```

Comment:
this can be rewritten to
else if (isUrlValid(url, fetchUrlFromProperty(FORMIO_URL))) {          
if (enableCustomSubmission && StringUtils.contains(url, "/submission")) {
                return CUSTOM_SUBMISSION_ACCESS_HANDLER;
} else {
		return FORM_ACCESS_HANDLER;
       }
}

Reply:
done